extensions/common/json-ld/src/test/java/org/eclipse/edc/jsonld/transformer/to/JsonObjectToPolicyTransformerTest.java

```diff
@@ -74,6 +76,8 @@ void setUp() {
         prohibition = Prohibition.Builder.newInstance().build();
         duty = Duty.Builder.newInstance().build();
 
+        target = "target";
```

Comment:
nit: use a constant

Reply:
I followed the convention already in place, not a fan of this but I didn't have the time to change everything